rdagent/log/time.py

```diff
@@ -0,0 +1,17 @@
+import time
+from functools import wraps
+
+
+def measure_time(method):
+    @wraps(method)
+    def timed(*args, **kwargs):
+        self = args[0]
+        start_time = time.time()
+        result = method(*args, **kwargs)
+        end_time = time.time()
+        duration = end_time - start_time
+        method_name = method.__name__
+        self.times[method_name] = duration
```

Comment:
check exist of self.times here? if not exist, self.times = {}. then delete `self.times={}` in outer class (RDLoop Class)